test/unit/floats.js

```diff
@@ -0,0 +1,82 @@
+var test = require('tap').test,
+nodeo = require(__dirname + '/../../lib/nodeo.js');
+
+var CF = nodeo.chromosome_float;
+var Classic = nodeo.classic_float;
+
+test('loads', function (t) {
+         t.ok(Classic, 'Loaded OK');
+         t.end();
+     });
+
+var population_size = 16,
+population = new Array([]),
+chromosome_size = 16;
+
+
+test('chromosomes floats', function (t) {
+    var tmpChr=[];
+    for( var i=0; i<chromosome_size; ++i ) tmpChr[i]=Math.random();
+    for ( var i = 0; i < population_size; i ++ )  {
+	var chromosome = new CF.Chromosome( tmpChr );
+	population.push( chromosome );
+	t.equal(chromosome.vector.length, chromosome_size, "Length " + chromosome.vector.toString() + " OK" );
```

Comment:
Line is too long.<br>Identifier 'chromosome_size' is not in camel case.<br>Mixed double and single quotes.